packages/openid4vc/src/openid4vc-issuer/router/accessTokenEndpoint.ts

```diff
@@ -176,6 +196,40 @@ export function handleTokenRequest(config: OpenId4VcIssuerModuleConfig) {
               }
             : undefined,
         })
+      } else if (grant.grantType === refreshTokenGrantIdentifier) {
```

Comment:
We might want to add an option whether to issue a refresh token, and how long the refresh token is valid.

Also do we issue a new refresh token every time you use a refresh token? In that case we also need to configure it, but maybe just issuing one refresh token that is valid for x time is sufficient?